src/libraries/Microsoft.PowerFx.Connectors/Tabular/Capabilities/ServiceCapabilities.cs

```diff
@@ -94,10 +94,15 @@ internal sealed class ServiceCapabilities : IColumnsCapabilities
         [JsonPropertyName(CapabilityConstants.SupportsRecordPermission)]
         public readonly bool SupportsRecordPermission;
 
+        [JsonInclude]
+        [JsonPropertyName(CapabilityConstants.SupportsJoin)]
+        public readonly bool SupportsJoinFunction;
```

Comment:
>SupportsJoinFunction [](http://example.com/codeflow?start=29&length=20)

does this imply we're getting through CDP?

Reply:
that's how a CDP connector could enable this capability
today we have none but I imagine CDP via shims could support it (not SN but another)

Reply:
So this is a proposed update to CDP contract?

Reply:
Very likely yes